services/web/server/src/simcore_service_webserver/folders/_folders_repository.py

```diff
@@ -31,18 +32,16 @@
 from simcore_postgres_database.utils_workspaces_sql import (
     create_my_workspace_access_rights_subquery,
 )
-from sqlalchemy import func
+from sqlalchemy import sql
```

Comment:
not anymore sa?

Reply:
`sa` is different to `sql`.

- The first is an alias for `sqlalchemy`
```python
import sqlalchemy as sa
```
- The second is an alias for `sqlalchemy.sql`, which includes all the sql expressions
```python
import sqlalchemy import sql
```

Reply:
ah ok no worries. was a misread